### ISSUE TITLE
FIXME: Fails to load mod config when all mods are disabled

### ISSUE THREAD

**Issue by DarkShadow44** [RimSort/RimSort#271]
**Describe the bug**
When disabling all mods (and possibly the core Rimwold mod) RimSort fails to load the modlist. It instead gives a message about some missing mods which random text as "mod ids".

**To Reproduce**
1. Remove all active mods, even the core mod.
2. Save mod list
3. Restart RimSort
4. An error message should now be displayed.

**Expected behavior**
It should work.

**Screenshots**
![Screenshot_20240229_192728](https://github.com/RimSort/RimSort/assets/2029892/c6022c57-67e9-4a2a-a9a6-eb656d27784a)

**Please complete the following information:**
 - OS: Arch Linux
 - RimSort version: Alpha-v1.0.6.2-hf
 - Please provide a `RimSort.log`: [RimSort.log](https://github.com/RimSort/RimSort/files/14452039/RimSort.log)
 - Github release

**Additional context**
[ModsConfig.zip](https://github.com/RimSort/RimSort/files/14452028/ModsConfig.zip) (Zip because I can't attach xml)

**Comment by twstagg** [comment]
Fixed in branch `instancing`
